spec/requests/moderations_spec.rb

```diff
@@ -135,4 +136,52 @@
       expect(response.body).to include "circle centered-icon adjustment-icon subtract"
     end
   end
+
+  describe "/mod" do
+    # rubocop:disable Layout/LineLength
+    let(:coc_guides_copy) { 'Check out our <a href="/code-of-conduct">Code of Conduct</a> and read through our <a href="/community-moderation">Trusted User Guide</a> and <a href="/tag-moderation">Tag Moderation Guide</a>.' }
```

Comment:
```suggestion
    let(:coc_guides_copy) { 
        'Check out our <a href="/code-of-conduct">Code of Conduct</a> and read through our <a href="/community-moderation">Trusted User Guide</a> and <a href="/tag-moderation">Tag Moderation Guide</a>.' 
     }
```
By the way, you could make this copy whatever you want since this is a test -- if you shorten it, you can remove the rubocop disable lines 😉